<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rectangle</class>
 <widget class="QWidget" name="Rectangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>宽度</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_b">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_h">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_h">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>高度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_M">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>弯矩</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_M">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Calculation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始计算（更新数据）</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>400</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Drawing">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>550</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始配筋</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_a_s">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>a_s</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_a_s">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保护层</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_n">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>根数</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_n">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_d">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_d">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>直径</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
